api/v1/prefectserver_types.go

```diff
@@ -281,7 +284,10 @@ func (s *PrefectServer) Image() string {
 }
 
 func (s *PrefectServer) Command() []string {
-	return []string{"prefect", "server", "start", "--host", "0.0.0.0"}
+	command := []string{"prefect", "server", "start", "--host", "0.0.0.0"}
+	command = append(command, s.Spec.ExtraArgs...)
```

Comment:
would this really be `extraCommands` since args are their own field in k8s?

Reply:
hmmm. technically yes, but here we're shoving args into the command field and not specifying any args directly, even though everything after the first one is an arg 🤔